experimental/PHP/Dockerfile

```diff
@@ -0,0 +1,47 @@
+﻿FROM microsoft/iis
+
+#
+# PHP 7.1 x64 running on IIS
+#
+
+ENV PHP C:\PHP
+
+# Install VC Redist 14
+ADD https://download.microsoft.com/download/6/A/A/6AA4EDFF-645B-48C5-81CC-ED5963AEAD48/vc_redist.x64.exe /vc_redist.x64.exe
+RUN C:\vc_redist.x64.exe /quiet /install
+
+# Install PHP
+RUN powershell.exe -Command Invoke-WebRequest 'https://sourceforge.net/projects/phpinstallermsi/files/zip/php-7.1.7-nts-Win32-VC14-x64.zip/download' -UserAgent '' -OutFile C:\php.zip
+RUN powershell -Command Expand-Archive -Path c:\php.zip -DestinationPath C:\PHP
+
+# Install PHP wincache
+RUN powershell.exe -Command Invoke-WebRequest 'https://sourceforge.net/projects/wincache/files/wincache-2.0.0/wincachewpi-2.0.0.8-7.1-nts-vc14-x64.exe/download' -UserAgent '' -OutFile C:\php_wincache.exe
+RUN C:\php_wincache.exe /Q /C /T:C:\php_wincache_msi
+RUN msiexec /a C:\php_wincache_msi\wincache71x64wpi.msi /qb TARGETDIR=C:\php_wincache_msi\extracted
+RUN copy C:\php_wincache_msi\extracted\PFiles\php_wincache.dll c:\PHP\ext

```

Comment:
If you don't need to install the msi, I'd recommend using a multi-stage if this is only used as a means to extract the dll